types/underscore/underscore-tests.ts

```diff
@@ -2109,6 +2109,9 @@ _.flatten([[[1, 2], [3]], [[4, 5]]]); // $ExpectType any[]
 // shallow flattening an array
 _.flatten([[[1, 2], [3]], [[4, 5]]], true); // $ExpectType number[][]
 
+// shallow flattening an array
+_.flatten([[[1, 2], [3]], [[4, 5]]], 2); // $ExpectType number[][]
```

Comment:
I think this should actually expect `number[]`. That might not pass, though. It's impossible to type `_.flatten` accurately over all possible flattening levels, which is why there is an `$ExpectType any[]` just a few lines up. Nevertheless, it is better to not keep this test as-is, because it is essentially untrue and we must not confuse future editors of these types.

A quick fix would be to just add another level of nesting to the test expression.

```suggestion
_.flatten([[[[1, 2], [3]], [[4, 5]]]], 2); // $ExpectType number[][]
```

Reply:
When I apply your suggested change, the return type changes to `number[][][]`. :thinking:

Reply:
Yes, because the current typing is treating any numerical `depth` arguments as if the depth is `1`.